src/Groups/types.jl

```diff
@@ -285,6 +291,11 @@ f1*f3
 """
 const SubPcGroupElem = BasicGAPGroupElem{SubPcGroup}
 
+function Base.hash(x::Union{PcGroupElem,SubPcGroupElem}, h::UInt)
+  G = full_group(parent(x))[1]
+  return hash(letters(x), hash(G, h))
```

Comment:
If we think of groups with large relative orders then `letters(x)` can be expected to be a long list.
`syllables(x)` would usually be shorter.

Both `letters` and `syllables` create new objects. Ideally we would take the internally stored exponent vector; for that, a GAP function analogous to `HashPermutation` would be needed.